### ISSUE TITLE
Test with third party cookies "From visited"

### ISSUE THREAD

**Issue by ianb** [mozilla-services/screenshots#3648]
Per https://github.com/mozilla-services/screenshots/issues/3480#issuecomment-337341302 we have a report of what appears to be an authentication issue with a user with third party cookies not fully disabled, but disabled in more limited cases. We should test if these cases also cause a problem. E.g., maybe it matters if your first interaction is before you have visited screenshots.firefox.com?

The way #3632 works, it will probably catch any cases that are problematic (including this and add-ons that mess with the cookies), but the message may give an inaccurate solution to the problem.

**Comment by wresuolc** [comment]
Ian also mentions this might be a Nightly thing

**Comment by SoftVision-CosminMuntean** [comment]
I have tested the Screenshots functionality with the "Accept third-party cookies" option set to "From visited". 

First I have tested this using latest Firefox 56.0 release with Screenshots 10.12.0 in order to see if I can reproduce the described issue in #3480. Unfortunately I haven't managed to reproduce it. Screenshots works as expected if the "Accept third-party cookies" option is set to "From visited". 
Tested using multiple scenarios on Mac Os, Windows and also on Linux.

I have also tested this on latest Nightly (58.0a1) build on Mac, Windows and Linux, but the issue is not reproducible. 
However, I have looked over the reporter's installed add-ons and it seems that he has the "mHeaderControl" addon installed. I can reproduce this issue if I'm using that add-on with certain configuration of it.
I have asked the reporter in #3480 if he can reproduce the issue with the add-on disabled.
